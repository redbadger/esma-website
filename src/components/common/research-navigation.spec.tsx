import React from "react";
import renderer from "react-test-renderer";
import ResearchNavigation from "./research-navigation";

describe("Research navigation", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ResearchNavigation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
