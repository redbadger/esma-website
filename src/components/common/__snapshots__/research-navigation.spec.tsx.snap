// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Research navigation renders correctly 1`] = `
<nav
  className="css-f47r1u-researchNavigationCss"
>
  <div
    className="scroll-help left can-scroll"
    onClick={[Function]}
    onKeyDown={[Function]}
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-chevron-left fa-w-10 "
      data-icon="chevron-left"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 320 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </div>
  <ul>
    <li
      className="early-years"
    >
      <a
        getProps={[Function]}
        href="/issues-chronology/early-years/"
      >
        <test-file-stub />
        <span>
          Early years
        </span>
      </a>
    </li>
    <li
      className="school-years"
    >
      <a
        getProps={[Function]}
        href="/issues-chronology/school-years/"
      >
        <test-file-stub />
        <span>
          School years
        </span>
      </a>
    </li>
    <li
      className="further-education"
    >
      <a
        getProps={[Function]}
        href="/issues-chronology/further-education/"
      >
        <test-file-stub />
        <span>
          Further education
        </span>
      </a>
    </li>
    <li
      className="higher-education"
    >
      <a
        getProps={[Function]}
        href="/issues-chronology/higher-education/"
      >
        <test-file-stub />
        <span>
          Higher education
        </span>
      </a>
    </li>
    <li
      className="working-life"
    >
      <a
        getProps={[Function]}
        href="/issues-chronology/working-life/"
      >
        <test-file-stub />
        <span>
          Working life
        </span>
      </a>
    </li>
  </ul>
  <div
    className="scroll-help right can-scroll"
    onClick={[Function]}
    onKeyDown={[Function]}
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-chevron-right fa-w-10 "
      data-icon="chevron-right"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 320 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </div>
</nav>
`;
